packages/abap-deploy-config-inquirer/test/service-provider-utils/transport-config.test.ts

```diff
@@ -193,7 +192,6 @@ describe('getTransportConfigInstance', () => {
             credentials: {}
         });
         expect(transportConfigResult3.transportConfigNeedsCreds).toBe(undefined);
-        expect(transportConfigResult3.warning).toContain('A certificate error has occurred Need help with this error?');
```

Comment:
With the removal of this, will need to check if the cert error GA link is still logged for CLI use.